net-vpn/headscale/headscale-0.15.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module systemd
+
+GENTOO_DEPEND_ON_PERL=no
+
+DESCRIPTION="An open source, self-hosted implementation of the Tailscale control server."
+HOMEPAGE="https://github.com/juanfont/headscale"
+SRC_URI="
+	https://github.com/juanfont/headscale/archive/refs/tags/v0.15.0.tar.gz -> ${P}.tar.gz
+	https://dev.gentoo.org/~jsmolic/distfiles/${P}-deps.tar.xz
+"
+LICENSE="BSD"
```

Comment:
Did you check deps for all bundled packages? You can use https://packages.gentoo.org/packages/dev-go/go-licenses to verify this